src/docs/develop/variables.md

```diff
@@ -2,27 +2,28 @@
 title: Variables
 ---
 
-Service variables are provided whenever you build, deploy, or run `railway run`. When
-defined, they are made available to your application at runtime as environment variables.
+Variables provide a powerful way to manage configuration and secrets across services in Railway.
 
-Variables are made available in the following scenarios:
+When defined, they are made available to your application as environment variables in the following scenarios:
 
 - The build process for each service deployment.
 - The running service deployment.
 - The command invoked by `railway run <COMMAND>`
 - The local shell via `railway shell`
 
-## Defining Variables
+In Railway, there is also a notion of configuration variables which allow you to control the behavior of the platform.
 
-Variables can be added via the Railway Dashboard by navigating to a service's "Variables" tab.
+
+## Service variables
+
+Define variables scoped to individual services by navigating to a service's "Variables" tab.
 
 <Image src="https://res.cloudinary.com/railway/image/upload/c_scale,w_2026/v1678820924/docs/CleanShot_2023-03-14_at_12.07.44_2x_rpesxd.png"
 alt="Screenshot of Variables Pane"
 layout="responsive"
 width={2026} height={933} quality={100} />
 
-You can view all variables for the current environment using the [Railway CLI](/develop/cli) with
-`railway variables` and change the environment with `railway environment`.
+Click on `New Variable` to enter your new variable into a form field or use the `RAW Editor`.
 
 ### Raw Editor
```

Comment:
Now, reading through, is the raw editor required in the docs, or can we get by without it? I'm unsure if we need the pictures as it kinda "breaks up" the logical flow

Reply:
I know what you mean, and I really think the restructure addresses the issue.  There should be a section that is mostly narrative - few to none pictures, no "this is how you do this" - and then we should have other pages (guides) where we put the images and helper information about how to do something and where you find things.

Reply:
for the sake of this PR though, I kinda think we should keep the image because it does help show people what we're talking about when we say "Raw Editor".  I'm okay with taking it out though if you think it's too distracting

Reply:
Love it. You're driving, so please just take my comments as random thoughts

Take em or leave em; totally fine either way!